### PR TITLE
Fix tinymce--missing fixes from 5.1/5.0

### PR DESCRIPTION
**Pull request by vangheem**
@mauritsvanrees towncrier integration is not working for me.

I keep getting `ValueError: Empty module name`.

Tried install globally and as part of buildout.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
Does that have anything to do with this PR?  You shouldn't have to use `towncrier` while creating a PR, although it is nice if you test that your news snippet renders the way you think it will.

Ah, I see. You are probably calling it like this in the `Products.CMFPlone` dir:

```
$ towncrier --draft
Loading template...
Finding news fragments...
Rendering news fragments...
Traceback (most recent call last):
...
  File ".../towncrier/__init__.py", line 47, in _main
    answer_yes
  File "...towncrier/__init__.py", line 98, in __main
    config['package'])
  File ".../towncrier/_project.py", line 46, in get_version
    module = _get_package(package_dir, package)
  File ".../towncrier/_project.py", line 25, in _get_package
    module = import_module(package)
  File ".../lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ValueError: Empty module name
```

`towncrier` must have a version number, so it tries to determine that here. Theoretically this can work if `pyproject.toml` contains a `package` key with value `Products.CMFPlone`, and that package is available on `sys.path`. Practically that is not going to work for most Plone packages. See https://github.com/hawkowl/towncrier/issues/111.

Anyway, if you supply an explicit version number, it will work:

```
$ towncrier --draft --version 1.0
Loading template...
Finding news fragments...
Rendering news fragments...
Draft only -- nothing has been written.
What is seen below is what would be written.

1.0 (2018-11-14)
----------------


No significant changes.
```

**Comment by vangheem** [comment]
I see, I guess I thought I was to use towncrier to generate the entry but I can just manually do it.

File: Products/CMFPlone/_scripts/_generate_gruntfile.py [hunk 1]
```diff
@@ -271,7 +271,7 @@ def resource_to_dir(resource, file_type='.js'):
 # LESS CONFIGURATION
 
 modify_vars = {}
-modify_vars['sitePath'] = '/'
+modify_vars['sitePath'] = "'/'"
```
Comment (mauritsvanrees) [inline Products/CMFPlone/_scripts/_generate_gruntfile.py]: This is in 5.0 but not in 5.1, so I guess this needs forward porting to 5.1 as well.
I don't really know if the change is good; I just compare versions.

File: Products/CMFPlone/profiles/dependencies/registry.xml [hunk 2]
```diff
@@ -623,8 +623,8 @@
       <value key="export">window.tinyMCE</value>
       <value key="init">function () { this.tinyMCE.DOM.events.domLoaded = true; return this.tinyMCE; }</value>
       <value key="css">
-        <element>++plone++static/components/tinymce-builded/js/tinymce/skins/lightgray/skin.less</element>
-        <element>++plone++static/components/tinymce-builded/js/tinymce/skins/lightgray/Content.less</element>
+        <element>++plone++static/components/tinymce-builded/js/tinymce/skins/lightgray/skin.min.css</element>
+        <element>++plone++static/components/tinymce-builded/js/tinymce/skins/lightgray/content.min.css</element>
```
Comment (mauritsvanrees) [inline Products/CMFPlone/profiles/dependencies/registry.xml]: I can confirm that these are needed.

**Comment by vangheem** [comment]
Test failure seems unrelated and I'm not sure how to restart a new test...

**Comment by ale-rt** [comment]
To restart the tests go to https://jenkins.plone.org/job/pull-request-5.2-3.6/ login and click "Build with Parameters" passing the PR link as usual. 
I already did that.
I think Py2.7 failure is related to this: https://github.com/plone/plone.app.widgets/commit/06eba491a6fc02391998977bff9fe92bc57fbc68#commitcomment-31510701

**Comment by mauritsvanrees** [comment]
Failure is unrelated, I will merge.

**Pull request merged**